### PR TITLE
Update reference to custom resource

### PR DESCRIPTION
**Pull request by zaro0508**
The cfn-sc-actions-provider moved and now deploys to a different bucket
path so we need to update the reference to the new location.

The custom resource moved to
https://github.com/Sage-Bionetworks-IT/cfn-cr-sc-actions-provider

There was a bug in the underlying lambda build,  it's fixed with PR
https://github.com/Sage-Bionetworks-IT/cfn-cr-sc-actions-provider/pull/2

This build depends on that PR.

### CODE REVIEW

File: config/prod/cfn-sc-actions-provider.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-template_path: remote/cfn-sc-actions-provider.yaml
+template_path: remote/cfn-cr-sc-actions-provider.yaml
 stack_name: cfn-sc-actions-provider
```
Comment (tthyer) [inline config/prod/cfn-sc-actions-provider.yaml:2]: Do you want to update the stack name to match the filename?

File: config/prod/cfn-sc-actions-provider.yaml [hunk 2]
```diff
@@ -8,4 +8,4 @@ dependencies:
   - prod/bootstrap.yaml
 hooks:
   before_launch:
-    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/cfn-sc-actions-provider.yaml --create-dirs -o templates/remote/cfn-sc-actions-provider.yaml"
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/cfn-cr-sc-actions-provider/master/cfn-cr-sc-actions-provider.yaml --create-dirs -o templates/remote/cfn-cr-sc-actions-provider.yaml"
```
Comment (tthyer) [inline config/prod/cfn-sc-actions-provider.yaml:11]: This is all correct but unfortunately will fail due to an error in your new repo: https://github.com/Sage-Bionetworks-IT/cfn-cr-sc-actions-provider/blob/master/.travis.yml#L33
Copies `.aws-sam/build/template.yaml`, but should be copying `.aws-sam/build/$REPO_NAME.yaml`.
The origin of this error may be in the lambda-template repo.

File: config/prod/cfn-sc-actions-provider.yaml (hunk 1 above)
Comment (zaro0508) [reply config/prod/cfn-sc-actions-provider.yaml:2]: no, there are `!ImportValue` dependencies on this stack name

File: config/prod/cfn-sc-actions-provider.yaml (hunk 2 above)
Comment (zaro0508) [reply config/prod/cfn-sc-actions-provider.yaml:11]: thanks for pointing that out, will fix.

Comment (zaro0508) [reply config/prod/cfn-sc-actions-provider.yaml:11]: ok, fixed with PR https://github.com/Sage-Bionetworks-IT/cfn-cr-sc-actions-provider/pull/2

**Pull request merged**